RefactoringsExperiments/src/main/java/org/jetbrains/research/extractMethodExperiments/CMDCaller.java

```diff
@@ -0,0 +1,82 @@
+package org.jetbrains.research.extractMethodExperiments;
+
+import org.apache.commons.cli.*;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.core.Logger;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.apache.logging.log4j.core.appender.ConsoleAppender;
+import org.apache.logging.log4j.core.config.Configuration;
+import org.apache.logging.log4j.core.config.ConfigurationFactory;
+import org.apache.logging.log4j.core.config.ConfigurationSource;
+import org.apache.logging.log4j.core.config.LoggerConfig;
+import org.apache.logging.log4j.core.config.xml.XmlConfigurationFactory;
+import org.apache.logging.log4j.core.layout.PatternLayout;
+
+import java.io.FileInputStream;
+
+public class CMDCaller {
+    /**
+     * Calls for `TrueRefactoringsExtractorCaller` and
+     * `FalseRefactoringsExtractorCaller` from command-line
+     */
+    public static void main(String[] args) throws Exception {
+
+        ConfigurationFactory factory = XmlConfigurationFactory.getInstance();
```

Comment:
i suggest moving the code responsible for logger configuration into a separate method and add its call here